druid-shell/src/windows/util.rs

```diff
@@ -280,3 +280,13 @@ fn attach_console() {
         }
     }
 }
+
+/// Returns the current locale string.
+///
+/// This should a [Unicode language identifier].
+///
+/// [Unicode language identifier]: https://unicode.org/reports/tr35/#Unicode_language_identifier
+pub fn get_locale() -> String {
+    //TODO ahem
```

Comment:
I believe the correct API here is [GetUserPreferredUILanguages function](https://docs.microsoft.com/en-us/windows/win32/api/winnls/nf-winnls-getuserpreferreduilanguages), but am not 100% sure. This function returns a list of languages, but in my testing I was only able to get one. For what we have now, I think just grabbing the first language is good.

Reply:
This would be annoying for me to do without access to a windows machine, so I'm going to leave this stubbed out for now.